pkg/restapi/operation/didexchange/didexchange_test.go

```diff
@@ -319,9 +318,28 @@ func TestServiceEvents(t *testing.T) {
 	didExSvc, err := didexsvc.New(&did.MockDIDCreator{}, &protocol.MockProvider{CustomStore: store})
 	require.NoError(t, err)
 
+	done := make(chan struct{})
+
 	// create the client
 	op, err := New(&mockprovider.Provider{StorageProviderValue: mockstore.NewMockStoreProvider(),
-		ServiceValue: didExSvc}, webhook.NewHTTPNotifier(nil))
+		ServiceValue: didExSvc},
+		&mockNotifier{
+			notifyFunc: func(topic string, message []byte) error {
+				require.Equal(t, "connections", topic)
+
+				conn := didexchange.Connection{}
+				jsonErr := json.Unmarshal(message, &conn)
+				require.NoError(t, jsonErr)
+
+				if conn.State == "responded" {
+					done <- struct {
+					}{}
```

Comment:
Not sure, but is there something funny going on with the curly braces here?

Reply:
It's passing the empty struct to the channel.

Reply:
It might just be me but I can't figure out what the extra {} is for

Reply:
This is a good reference - https://stackoverflow.com/questions/45122905/how-do-struct-and-struct-work-in-go.

Note - I have updated the code to close the channel rather than passing the struct.